src/test/resources/features/refapp-2.x/06-user/users.feature

```diff
@@ -1 +1,31 @@
-Feature: Users Management
\ No newline at end of file
+Feature: User Account Management
+
+  Background:
+    Given a user clicks on system administartion app from home page
+    Then the system loads system administration page
+    When a user clicks on manage accounts app
+    Then the system loads manage acccount page
+
+  @selenium
+  @userAccount
+  Scenario: Create user accounts
+    # User story: Creating user account for Data clerk
+    When a user clicks add new account button
+    Then the system loads add new account form
+    And a user enters data clerk details in the user account form
+    And a user clicks on save account button
+    Then the system adds user account into the users table
+
+    # User story: Creating user account for Nurse
+    When a user clicks add new account button
```

Comment:
There is still 
```suggestion
    When a user clicks add new account button
```
more than once, is it possible to only have that scenario once .this will improve code readability